index.html

```diff
@@ -2,9 +2,181 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
-	<title>Title</title>
+	<link rel="stylesheet" href="styles.css">
+	<title>Rekupe - Digital Agency in LA</title>
 </head>
 <body>
+<header class="header__outer-container">
+	<div class="header__inner-container">
+		<nav class="navigation">
+			<ul class="navigation__list">
+				<li class="navigation__list-items"><a class="navigation___link" href="#main__content-section">case studies</a></li>
+				<li class="navigation__list-items"><a class="navigation___link" href="#main__services--inner-container">services</a></li>
+				<li class="navigation__list-items"><a class="navigation___link" href="#footer__outer-container">contact us</a></li>
+			</ul>
+		</nav>
+		<div class="header__hero-section">
+			<h1>your website is an investment in the success of your business</h1>
+			<p class="header__hero-section-paragraph">We are here to help you succeed.</p>
+			<a href="#footer__outer-container">
+				<button class="main__cta-button">let's work together</button>
+			</a>
+		</div>
+		<div class="header__squiggle-line">
```

Comment:
Dit kon je ook doen door bijvoorbeeld alleen een <div> te declareren en die een class te geven en vervolgens de styling met het SVG in de CSS te doen :)